SQL.js

```diff
@@ -17,28 +17,19 @@ class SqlStatement {
 
   glue (pieces, separator) {
     const result = { strings: [], values: [] }
-
     let carryover
     for (let i = 0; i < pieces.length; i++) {
       const strings = Array.from(pieces[i].strings)
-      if (typeof carryover === 'string') {
+      if (i > 0) {
         strings[0] = carryover + separator + strings[0]
         carryover = null
```

Comment:
is there a reason to set `carryover` to null here? as the next row is already setting the `carryover`

Reply:
There is no reason, thank you!